src/utils/components/DiskModal/components/StorageProfileSettings/ApplyStorageProfileSettings.tsx

```diff
@@ -1,79 +1,57 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
 import { useFormContext } from 'react-hook-form';
 
-import { V1beta1StorageSpecVolumeModeEnum } from '@kubevirt-ui/kubevirt-api/kubevirt';
-import ApplyStorageProfileSettingsCheckbox from '@kubevirt-utils/components/ApplyStorageProfileSettingsCheckbox/ApplyStorageProfileSettingsCheckbox';
+import ErrorAlert from '@kubevirt-utils/components/ErrorAlert/ErrorAlert';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import useStorageProfileClaimPropertySets from '@kubevirt-utils/hooks/useStorageProfileClaimPropertySets';
-import { isEmpty } from '@kubevirt-utils/utils/utils';
-import { Flex, FlexItem } from '@patternfly/react-core';
+import { Checkbox, Flex, FlexItem, Skeleton } from '@patternfly/react-core';
 
 import { V1DiskFormState } from '../../utils/types';
-import {
-  ACCESS_MODE_FIELD,
-  DATAVOLUME_TEMPLATE_STORAGE,
-  STORAGE_PROFILE_SETTINGS_APPLIED_FIELD,
-  VOLUME_MODE_FIELD,
-} from '../utils/constants';
+import { DATAVOLUME_TEMPLATE_STORAGE } from '../utils/constants';
 
 import AccessMode from './AccessMode';
 import VolumeMode from './VolumeMode';
 
 import './ApplyStorageProfileSettings.scss';
 
 const ApplyStorageProfileSettings: FC = () => {
-  const { setValue, watch } = useFormContext<V1DiskFormState>();
+  const { t } = useKubevirtTranslation();
+  const [allowAllModes, setAllowAllModes] = useState<boolean>(false);
+  const { watch } = useFormContext<V1DiskFormState>();
 
-  const [storageProfileSettingsApplied, storage] = watch([
-    STORAGE_PROFILE_SETTINGS_APPLIED_FIELD,
-    DATAVOLUME_TEMPLATE_STORAGE,
-  ]);
+  const [storage] = watch([DATAVOLUME_TEMPLATE_STORAGE]);
+  const { storageClassName } = storage;
 
-  const { accessModes, storageClassName, volumeMode } = storage;
+  const { claimPropertySets, error, loaded } = useStorageProfileClaimPropertySets(storageClassName);
 
-  const { claimPropertySets, loaded: storageProfileLoaded } =
-    useStorageProfileClaimPropertySets(storageClassName);
+  if (!loaded) {
+    return <Skeleton screenreaderText={t('Loading Storage Profile')} />;
```

Comment:
in CNV we have a convention that a CR names like StorageProfile as PascalCase meaning Storage Profile should be presented as StorageProfile.

Reply:
OK. I had a similar case with InstanceType and I was told to change it back to "Instance Type".
CC: @metalice